### PR TITLE
TEST button returns full list

### PR DESCRIPTION
**Pull request by iloveMonday**
made function fetchTracks to... fetch tracks.
called it in the useEffect function and handleTestClick function

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
   "license": "ISC",
   "dependencies": {
     "express": "^4.18.2",
+    "normalize.css": "^8.0.1",
```
Comment (JarrettD5309) [inline package.json]: What is this dependency? Is it being used somewhere?

File: client/App.tsx [hunk 2]
```diff
@@ -9,17 +9,21 @@ const App = (): ReactElement => {
...
-      .catch((e) => console.log(e));
+    fetchTracks();
 
     fetch(`${URL}/genres`)
       .then((res) => res.json())
       .then((json) => setGenres(json))
       .catch((e) => console.log(e));
   }, []);
 
+  const fetchTracks = () => {
```
Comment (JarrettD5309) [inline client/App.tsx:20]: Looks good!

File: client/App.tsx [hunk 3]
```diff
@@ -37,14 +41,15 @@ const App = (): ReactElement => {
   const handleTestClick = function (): void {
     console.table(tracks);
     console.log(genres);
+    fetchTracks();
```
Comment (JarrettD5309) [inline client/App.tsx]: Obviously right now we are just using this button to console.log some data and to be able to test some functionality. It won't be part of the actual app. But I just wanted to point out that by running the `console.table(tracks)` before the `fetchTracks()` we will likely see what was saved to the `tracks` variable just before we set them to something new.

File: client/App.tsx [hunk 4]
```diff
@@ -37,14 +41,15 @@ const App = (): ReactElement => {
...
   };
 
 
 
   return (
     <>
       <h1>DJ Database</h1>
-      <button onClick={handleTestClick}>TEST</button>
+      <button onClick={(handleTestClick)}>TEST</button>
```
Comment (JarrettD5309) [inline client/App.tsx]: Is there a reason these parentheses were added here?

**Review by JarrettD5309 — COMMENTED** [review]
Wondering about the dependency that was added to the `package.json`. This should probably be removed unless there is a reason for it.

File: package.json (hunk 1 above)
Comment (iloveMonday) [reply package.json]: ah, normalize.css is a sorta css reset I installed the other night, turned it off for this.

File: client/App.tsx (hunk 4 above)
Comment (iloveMonday) [reply client/App.tsx]: My only guess is a new extension I installed in VSCode changing it. Trying out extensions as they're suggested in my class, I end up turning off 80% of them.

**Comment by iloveMonday** [comment]
Took out the normalize.css and parantheses, reordered handleTestClick.

**Review by JarrettD5309 — APPROVED** [review]

**Pull request merged**